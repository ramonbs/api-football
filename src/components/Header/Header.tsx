import style from "./header.module.scss";

function Header() {
  return (
    <header className={ style.header_container }>
        <h1>
            Seja bem vindo ao Meu Time!
        </h1>
    </header>
  )
}

export default Header
