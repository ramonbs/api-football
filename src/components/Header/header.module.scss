.header_container {
    max-width: 30rem;
    margin-inline: auto;
}