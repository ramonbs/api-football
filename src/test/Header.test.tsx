import Header from "../components/Header";
import { render, screen } from "@testing-library/react";

describe("Header", () => {
    it("should render correctly", () => {
        render(<Header />);
        expect(screen.getByText("Test")).toBeInTheDocument();
    });
});
