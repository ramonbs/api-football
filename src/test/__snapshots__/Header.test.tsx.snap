// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > should render correctly 1`] = `
<div>
  <header>
    <h1>
      Test
    </h1>
  </header>
</div>
`;
